Sources/JWTKeychain/Support/UserAuthenticating.swift

```diff
@@ -65,39 +71,38 @@ public class UserAuthenticator: UserAuthenticating {
         }
 
         let name = data[User.Keys.name]?.string
-        let email = data[User.Keys.email]?.string
 
-        return try user.update(
-            email: email.map(Valid.init),
+        try user.update(
+            email: nil,
```

Comment:
This should be changed back. It had to do with an earlier version where I fetched user by email and therefore email would alway be present. Now it will only be present in case it needs updating.